client/web/src/integration/profile.test.ts

```diff
@@ -68,3 +72,52 @@ describe('User profile page', () => {
         assert.strictEqual(requestVariables.displayName, 'Test2')
     })
 })
+
+describe('User Different Settings Page', () => {
+    let driver: Driver
+    before(async () => {
+        driver = await createDriverForTest()
+    })
+    after(() => driver?.close())
+    let testContext: WebIntegrationTestContext
+    beforeEach(async function () {
+        testContext = await createWebIntegrationTestContext({
+            driver,
+            currentTest: this.currentTest!,
+            directory: __dirname,
+        })
+    })
+    afterEachSaveScreenshotIfFailed(() => driver.page)
+    afterEach(() => testContext?.dispose())
+
+    it('display user email setting page', async () => {
+        testContext.overrideGraphQL({
+            ...commonWebGraphQlResults,
+            UserAreaUserProfile: () => ({
+                user: USER,
+            }),
+            UserSettingsAreaUserProfile: () => ({
+                node: USER,
+            }),
+        })
+        await driver.page.goto(driver.sourcegraphBaseUrl + '/users/test/settings/email')
+        await percySnapshotWithVariants(driver.page, 'User Email Settings Page')
+        await accessibilityAudit(driver.page)
+    })
+
+    it('display user password setting page', async () => {
+        testContext.overrideGraphQL({
+            ...commonWebGraphQlResults,
+            UserAreaUserProfile: () => ({
+                user: USER,
+            }),
+            UserSettingsAreaUserProfile: () => ({
+                node: USER,
+            }),
+        })
+        await driver.page.goto(driver.sourcegraphBaseUrl + '/users/test/settings/password')
```

Comment:
```suggestion
        await driver.page.goto(driver.sourcegraphBaseUrl + '/user/settings/password')
```

The current page shows a version that isn't seen my most pages (a site admin error). See [Percy snapshot](https://percy.io/Sourcegraph/Sourcegraph/builds/17641505/changed/989615826?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1920&widths=1920).

We want to show the page that most users show, which should be loaded from this URL I think (opens page for current user). It should look like this example:

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/9516420/165747370-eb62036c-0926-4efe-835c-eca948f472b8.png">